sherpa/astro/plot.py

```diff
@@ -141,17 +138,17 @@ def prepare(self, data, stat=None):
         _check_hist_bins(self)
 
 
-class ModelPHAHistogram(HistogramPlot):
+class ModelPHAHistogram(shplot.HistogramPlot):
```

Comment:
I think that separating out the sherpa.plot and sherpa.astro.plot namespaces, as you've done here, is a good idea as the naming of the different classes is "not great"™ so explicitly having to identify where the name comes from should help in the long run.